data/src/test/java/org/apache/iceberg/data/GenericAppenderHelper.java

```diff
@@ -45,34 +44,19 @@ public class GenericAppenderHelper {
 
   private final Table table;
   private final FileFormat fileFormat;
-  private final TemporaryFolder tmp;
-  private final Path temp;
+  private final Path tempDir;
```

Comment:
Is there a particular reason to name this `tempDir` everywhere in the PR rather than just keeping it `temp`?